src/parser.js

```diff
@@ -97,15 +97,15 @@ export default class Parser {
       const resource = new Resource();
       resource.href = this.parseURL(section.resources[0].url);
 
-      for (const action of section.resources) {
+      section.resources.forEach((action) => {
```

Comment:
The Airbnb style guide is recommending to use pure high order functions which return values instead of having side effects. This `forEach` loop still has a side-effect of mutating the resource so I do not think it follows the style guide either.

> "Why? This enforces our immutable rule. Dealing with pure functions that return values is easier to reason about than side effects." -- https://github.com/airbnb/javascript#iterators-and-generators

This loop still has a side effect of pushing content into the `resource`. This could be solved by using map or something like:

```js
resource.content = section.resources.map((action) => {
    ...
    return transition;
});
```